src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -65,8 +65,8 @@ def clone(self):
         '''
         Clone mozilla-central
         '''
-        self.repo_dir = os.path.join(self.cache_root, 'central')
-        shared_dir = os.path.join(self.cache_root, 'central-shared')
+        self.repo_dir = os.path.join(self.cache_root, 'sa-central')
```

Comment:
Why the renaming?

Reply:
Just to get the project name in there "just in case". This has no real purpose other than being a bit more identifiable (i have several clones on my dev computer for uplifts & static analysis).